cli/src/commands/subgraph/commands/publish.ts

```diff
@@ -95,7 +95,9 @@ export default (opts: BaseCommandOptions) => {
 
     switch (resp.response?.code) {
       case EnumStatusCode.OK: {
-        spinner.succeed('Subgraph published successfully.');
+        spinner.succeed(
+          resp?.hasChanged === false ? 'Detected no new changes to publish.' : 'Subgraph published successfully.',
```

Comment:
"Skip publish, no changes to publish"

Reply:
Done, confirmed it doesnt cause any compositions